### PR TITLE
[plugin.video.nhlgcl@matrix] 2021.10.1+matrix.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Add-on details:

- **General**
  - Add-on name: NHL TV™
  - Add-on ID: plugin.video.nhlgcl
  - Version number: 2021.10.1+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/eracknaphobia/plugin.video.nhlgcl
  
With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi
        

### Description of changes:

            - Add Seattle to Favorite Team selection
            - Refactor Fav Team code
            - Build game images dynamically
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Review by basrieter — CHANGES_REQUESTED** [review]
Same comments as https://github.com/xbmc/repo-plugins/pull/3741

**Comment by eracknaphobia** [comment]
@basrieter I've made the changes you requested. I'd appreciate a review when you get a chance.

File: plugin.video.nhlgcl/resources/lib/globals.py [hunk 1]
```diff
@@ -55,6 +58,7 @@
 FREE = 'FF43CD80'
 
 ROOTDIR = xbmcaddon.Addon().getAddonInfo('path')
+USER_DATA_DIR = os.path.join(xbmc.translatePath("special://userdata"), 'addon_data', ADDON_ID)
```
Comment (enen92) [inline plugin.video.nhlgcl/resources/lib/globals.py:61]: xbmc.translatePath was already removed in nexus. Can you use xbmcvfs translatePath instead?

**Review by enen92 — CHANGES_REQUESTED** [review]
Just a small thing that needs changing and it's good to go

File: plugin.video.nhlgcl/resources/lib/globals.py (hunk 1 above)
Comment (eracknaphobia) [reply plugin.video.nhlgcl/resources/lib/globals.py:61]: I believe line 19 thru 22 should handle the conversion.

Comment (enen92) [reply plugin.video.nhlgcl/resources/lib/globals.py:61]: You're right, didn't spot that in the phone. Redefining Kodi built-in module behaviour is not a good practice though. Consider wrapping this into a function at some point in the future

**Review by enen92 — APPROVED** [review]

**Pull request merged**